### DISCUSSION TITLE
Updates for mroonga

### DISCUSSION THREAD

**Discussion by josepsanzcamp** [mroonga/mroonga#675] [General]
Hi team.

I have detected that appear new releases for groonga but not for mroonga.

Are you planning to stop updating the mroonga engine???

Thanks in advance.

Josep.

**Comment by kou** [comment]
We'll update Mroonga in a few months!
We need to follow MySQL/MariaDB changes before it!

**Reply by josepsanzcamp** [reply]
Ok.

Thanks in advance.

Josep.
